doozerlib/source_modifications.py

```diff
@@ -79,12 +79,14 @@ def __init__(self, *args, **kwargs):
         :param path: Destination path to the dist-git repo.
         :param overwriting: True to allow to overwrite if path exists.
           Setting to false to prevent from accidently overwriting files from in-tree source.
+        :param doozersource: Destination path to the doozer source
```

Comment:
Think we split words with `_` if an attribute has multiple words.

Reply:
Note that this attribute is not used in https://github.com/openshift-eng/ocp-build-data/pull/3356